### DISCUSSION TITLE
Json to Coco Conversion for Object Detection

### DISCUSSION THREAD

**Discussion by salvatoresarnelli** [bdd100k/bdd100k#320] [Q&A]
Hello, 
I have to perform the conversion of labels for Object Detection from Json format to Coco format. I followed the script provided by the documentation, but I have the following error: 

![Screenshot 2023-08-17 120723](https://github.com/bdd100k/bdd100k/assets/94001584/ea5217e5-f513-464f-ba81-f674b33c4f00)

In this ways the conversion fails. Any help please? 
Of course I've used the annotations file provided from the official website.

**Comment by SiBensberg** [comment]
I fixed it with downgrading `pydantic`. The scalabel code seems to be incompatible with the new pydantic 2.0 release. Downgrading to 1.10.12 fixed it for me.
```pip install pydantic==1.10.12```

**Reply by salvatoresarnelli** [reply]
I love you.
